index.ts

```diff
@@ -63,7 +66,9 @@ export const main = async (): Promise<void> => {
   const mfest = manifest(name, buildNumber);
   const manifestJSON = JSON.stringify(mfest);
 
-  const stagingDir = "staging";
+  // Ensure unique name as multiple steps may run using this action within the
+  // same workflow (this has happened!).
+  const stagingDir = stagingDirOverride || fs.mkdtempSync("staging-");
```

Comment:
I don't _think_ it ends up mattering here but looking at the docs I think `mkdtempSync` returns a fully qualified/absolute directory path, which is different than before where `stagingDir` was always a single (relative) directory name.

Reply:
👍 I think this is fine, but you're right there is a difference here.